packages/lesswrong/components/search/SearchPageTabbed.tsx

```diff
@@ -343,8 +343,8 @@ const SearchPageTabbed = ({classes}:{
           defaultRefinement={true}
         />}
         {tab === 'Tags' && <ToggleRefinement
-          attribute="isSubforum"
-          label="Has subforum"
+          attribute="core"
+          label="Core topic"
```

Comment:
I think "Core topic" is a bit clearer than just "Core", even though it's kind of redundant. Open to changing this though